view/main/index.pug

```diff
@@ -1,5 +1,5 @@
-main.main
-  button.main_expand( aria-label="Expand or collapse panel" type="button" )
+main.main.layout_main#main-panel
+  button.main_expand( aria-label="Main panel" type="button" aria-expanded='true' aria-controls="main-panel" )
```

Comment:
Why we change label? The new label doesn't describe the action

Reply:
`aria-expanded` already say if element is collapsed or expanded. Screen reader even notify user on attribute change.

So with old label it sounds like oiled oil _(yeah, I didn't found English idiom)_

PS It's supposed to notify only that the [status has changed](https://doka.guide/a11y/aria-expanded/#:~:text=%D0%9A%D0%BE%D0%B3%D0%B4%D0%B0%20%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D1%82%D0%B5%D0%BB%D1%8C%20%D1%80%D0%B0%D1%81%D0%BA%D1%80%D0%BE%D0%B5%D1%82%20%D1%81%D0%BF%D0%B8%D1%81%D0%BE%D0%BA%2C%20%D1%81%D0%BA%D1%80%D0%B8%D0%BD%D1%80%D0%B8%D0%B4%D0%B5%D1%80%20%D0%BF%D1%80%D0%BE%D1%81%D1%82%D0%BE%20%D1%81%D0%BE%D0%BE%D0%B1%D1%89%D0%B8%D1%82%20%C2%AB%D1%80%D0%B0%D0%B7%D0%B2%D1%91%D1%80%D0%BD%D1%83%D1%82%D0%BE%C2%BB%20%D0%B1%D0%B5%D0%B7%20%D0%BF%D0%BE%D0%B2%D1%82%D0%BE%D1%80%D0%B5%D0%BD%D0%B8%D1%8F%20%D0%B8%D0%BD%D1%84%D0%BE%D1%80%D0%BC%D0%B0%D1%86%D0%B8%D0%B8%20%D0%BE%20%D0%BA%D0%BD%D0%BE%D0%BF%D0%BA%D0%B5), but on my mac, Voice Over is re-reading the entire message:
> Main panel, expanded, button
> Main panel, collapsed, button

Reply:
I will ask in a11y chat how people deal with such cases

Reply:
> Main panel, expanded, button
> Main panel, collapsed, button

Good note about `expanded`/`collapsed`. But I still prefer verb there.

Maybe `Toggle main panel`?

Reply:
Also, let’s remove `main` since this word doesn’t say anything

Reply:
Change to `toggle`.

PS Miss @glafirazhur confirmed that it's ok when a button in such contexts doesn't have a action label. For example, [accordion pattern](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/)